### PR TITLE
Multiple sheets source with latest changes

### PR DESCRIPTION
**Pull request by mikel3361**

### CODE REVIEW

File: src/cBiff12Writer.cls [hunk 1]
```diff
@@ -130,6 +136,21 @@ Public Function Init( _
     Init = True
 End Function
 
+Public Function AddSheet(Optional ByVal SheetName As String, Optional ByVal lColCount As Long) As Boolean
+    pvEndSheet m_oSheetPart
+    Set m_oSheetPart = m_oContainer.SheetPart(m_oContainer.SheetPartCount + 1)
+    m_oSheetPart.SheetName = IIf(Not IsMissing(SheetName), SheetName, "Sheet" & Format(m_oContainer.SheetPartCount, "0"))
+    '--- reset previous variables
+    m_lRowCount = 0
+    m_lColCount = Iif(Not IsMissing(lColCount), lColCount, 1)
```
Comment (wqweto) [inline src/cBiff12Writer.cls]: Using `IsMissing` for `Long` optionals is not possible.

Can we just keep current `m_lColCount` when `lColCount` is not supplied with a simple `If` statement?

File: src/cBiff12Writer.cls [hunk 2]
```diff
@@ -130,6 +136,21 @@ Public Function Init( _
     Init = True
 End Function
 
+Public Function AddSheet(Optional ByVal SheetName As String, Optional ByVal lColCount As Long) As Boolean
+    pvEndSheet m_oSheetPart
+    Set m_oSheetPart = m_oContainer.SheetPart(m_oContainer.SheetPartCount + 1)
+    m_oSheetPart.SheetName = IIf(Not IsMissing(SheetName), SheetName, "Sheet" & Format(m_oContainer.SheetPartCount, "0"))
```
Comment (wqweto) [inline src/cBiff12Writer.cls]: Using `IsMissing` for `String` optional param is not possible (works only w/ `Variant` params).

A `IIf(LenB(SheetName) = 0, ...` can be used for the intended optional string not supplied check.

Comment (mikel3361) [reply src/cBiff12Writer.cls]: I need  my IDE  ...
I check thus tomorrow at work and then we can merge

**Comment by mikel3361** [comment]
Update AddSheet und Init for SheetName with functions from your Info's Can test it tomorow or you make a test. Wenn this is ok you can merge it.

**Comment by wqweto** [comment]
LGTM

**Comment by wqweto** [comment]
Closes #1

**Pull request merged**